docs/tutorials/basic/data.md

```diff
@@ -4,6 +4,46 @@ Most training and inference modules in MXNet accept data iterators,
 which simplifies this procedure, especially when reading large datasets.
 Here we discuss the API conventions and several provided iterators.
 
+## Prerequisites
+
+To complete this tutorial, we need:  
+
+- [Python](https://www.python.org/downloads/)  
+
+- [PIP](https://pypi.python.org/pypi)  
+
+- [MXNet](http://mxnet.io/). You can either [build from source](http://mxnet.io/get_started/install.html) or install from [pip](https://pypi.python.org/pypi/mxnet).  
+
+```bash
+$ pip install mxnet
+```  
+- [OpenCV](http://opencv.org/opencv-3-2.html)  
+
+```bash
+$ pip install opencv-python
+```
+- [Python Requests package](http://docs.python-requests.org/en/master/)  
+
+```bash
+$ pip install requests
+```
+- [Matplotlib](https://matplotlib.org/)  
+
+```bash
+$ pip install matplotlib
+```
+- [Jupyter Notebook](http://jupyter.org/index.html)
```

Comment:
Installation page does not list all the dependencies(typically any dependency used within MXNet should either come from the PIP package or be a part of installation steps/script). some of the dependencies are only needed for the tutorial.